res/css/views/rooms/_EventTile.scss

```diff
@@ -962,8 +962,17 @@ $threadInfoLineHeight: calc(2 * $font-12px); // See: _commons.scss
         }
 
         &[data-layout=bubble] {
-            margin-inline-start: var(--BaseCard_EventTile-spacing-horizontal);
-            margin-inline-end: var(--BaseCard_EventTile-spacing-horizontal);
+            margin-inline-start: var(--BaseCard_EventTile-spacing-inline);
+            margin-inline-end: var(--BaseCard_EventTile-spacing-inline);
+
+            &::before {
+                inset-inline: calc(-1 * var(--BaseCard_EventTile-spacing-inline));
```

Comment:
`inset-inline` is always horizontal unless the UI would be vertical for Chinese, Japanese, or Mongolian languages :-D

Reply:
:+1: